### PR TITLE
Add old profile reminders

### PR DESCRIPTION
**Pull request by liujian1368928**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
提示开发 升级到2.2 版本 没有更改配置文件名称导致 失效问题

### CODE REVIEW

File: src/SkyApm.Agent.Hosting/Extensions/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -40,11 +40,13 @@
 using SkyApm.PeerFormatters.SqlClient;
 using SkyApm.PeerFormatters.MySqlConnector;
 using ILoggerFactory = SkyApm.Logging.ILoggerFactory;
+using System.IO;
 
 namespace Microsoft.Extensions.DependencyInjection
 {
     public static class ServiceCollectionExtensions
     {
+        private static string oldConfigName = "skywalking.json";
```
Comment (wu-sheng) [inline src/SkyApm.Agent.Hosting/Extensions/ServiceCollectionExtensions.cs:49]: When did we use this name? From which version?

**Comment by wu-sheng** [comment]
Please rename the title of PR to English.

File: sample/SkyApm.Sample.Logging/SkyApm.Sample.Logging.csproj [hunk 2]
```diff
@@ -2,7 +2,7 @@
     <PropertyGroup>
         <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
         <TargetFramework>net7.0</TargetFramework>
-        <Nullable>enable</Nullable>
```
Comment (wu-sheng) [inline sample/SkyApm.Sample.Logging/SkyApm.Sample.Logging.csproj:5]: What does this mean?

File: src/SkyApm.Agent.Hosting/Extensions/ServiceCollectionExtensions.cs (hunk 1 above)
Comment (liujian1368928) [reply src/SkyApm.Agent.Hosting/Extensions/ServiceCollectionExtensions.cs:49]: skywalking.json？？？

Comment (wu-sheng) [reply src/SkyApm.Agent.Hosting/Extensions/ServiceCollectionExtensions.cs:49]: Yes. 2.1 to 2.2?

Comment (liujian1368928) [reply src/SkyApm.Agent.Hosting/Extensions/ServiceCollectionExtensions.cs:49]: It used to be this sywalk.json, I used the old version to upgrade to 2.2, but apache-skywalk-APM-9.5.0 view is empty, but I have dropped to 2.1, view the source code only to find that the configuration name has changed

File: sample/SkyApm.Sample.Logging/SkyApm.Sample.Logging.csproj (hunk 2 above)
Comment (liujian1368928) [reply sample/SkyApm.Sample.Logging/SkyApm.Sample.Logging.csproj:5]: [enable](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-reference-types)

**Comment by liuhaoyang** [comment]
> builder.AddJsonFile("skywalking.json", true) 
Now it still supports configuring the file to be named as skywalking.json

**Comment by wu-sheng** [comment]
@liuhaoyang tole me this is a bug introduced by a previous commit. Let's wait for him about what should be the better fix.

**Comment by liujian1368928** [comment]
Sorry, I apologize for my lack of rigor. After careful testing, there is no problem

**Pull request closed without merging**